bentoml/_internal/bento/build_config.py

```diff
@@ -110,28 +107,29 @@ def _envars_validator(
     attribute: "Attribute[t.Union[str, t.List[str], t.Dict[str, t.Any]]]",
     value: t.Union[str, t.List[str], t.Dict[str, t.Any]],
 ) -> None:
-    if isinstance(value, str):
-        if not os.path.exists(value) and os.path.isfile(value):
-            raise InvalidArgument(
-                f'`env="{value}"` is not a valid path. if `env` is parsed as a string, it should be a path to an ".env" file.'
-            )
-
-    envars_rgx = re.compile(r"(^[A-Z0-9_]+)(\=)(.*\n)")
-
     if isinstance(value, list):
-        for envar in value:
-            match = envars_rgx.match(envar)
-            if match is None:
-                raise InvalidArgument(f"{envar} doesn't follow format ENVAR=value")
-
-            k, _ = envar.split("=")
-            if not k.isupper():
+        for envar in value:  # pragma: no cover
+            #  we have tests for this cases, but pytest refuse to cover
```

Comment:
it's ok if pytest fail to report the test coverage here, let's just report the issue upstream. no need to comment about it here.

Reply:
I concur; we probably shouldn't worry about pragma no cover until we're at 100% coverage.